src/app/shared/directives/create-workplace/is-this-your-workplace/is-this-your-workplace.directive.ts

```diff
@@ -67,7 +67,10 @@ export class IsThisYourWorkplaceDirective implements OnInit, AfterViewInit {
   }
 
   protected prefillForm(): void {
-    if (this.workplaceInterfaceService.selectedLocationAddress$.value.locationId === this.locationData.locationId) {
+    if (
+      this.workplaceInterfaceService.selectedLocationAddress$.value &&
+      this.workplaceInterfaceService.selectedLocationAddress$.value.locationId === this.locationData.locationId
+    ) {
```

Comment:
You can use typescript to make this easier

```suggestion
    if (
      this.workplaceInterfaceService.selectedLocationAddress$.value?.locationId === this.locationData.locationId
    ) {
```

Reply:
refactored in line with the above